README.md

```diff
@@ -131,6 +131,45 @@ builder.add_sequence(rc_dna_seq)
 
 If modifications or manually defined IDs are required, the complementary sequence must be created separately.
 
+#### Description (optional, JSON-only comment)
+
+AlphaFold3 added the option of specifying a textual description for protein, RNA and DNA chains and ligands. In af3cli, each sequence entity can carry an optional description string. This field is only used in the JSON format and serves as a human-readable comment describing the chain. If not set, it is omitted from the JSON.
```

Comment:
There should be a warning for the users, since it is only supported in version 4.